tests/components/closed-caption/closedCaptionPopover-test.js

```diff
@@ -81,38 +81,66 @@ describe('ClosedCaptionPopover', function() {
     };
 
     beforeEach(function() {
-      var DOM = TestUtils.renderIntoDocument(<ClosedCaptionPopover {...props} />);
-      popover = TestUtils.findRenderedDOMComponentWithClass(DOM, 'oo-popover-horizontal');
-      popoverMenuItems = popover.querySelectorAll('[' + CONSTANTS.KEYBD_FOCUS_ID_ATTR + ']');
+      wrapper = Enzyme.mount(<ClosedCaptionPopover {...props} />);
+      popover = wrapper.find('.oo-popover-horizontal');
+      popoverMenuItems = popover.getDOMNode().querySelectorAll('[' + CONSTANTS.KEYBD_FOCUS_ID_ATTR + ']');
     });
 
     afterEach(function() {
-      document.activeElement = null;
+      if (document.activeElement) {
+        document.activeElement.blur();
+      }
     });
 
+    //TODO: I couldn't find an easy way to simulate the keydown event with the proper event target,
+      //so I'm calling onKeyDown manually for the below tests for now
     it('should focus on previous menu item when pressing UP or LEFT arrow keys', function() {
       var activeIndex = popoverMenuItems.length - 1;
-      document.activeElement = popoverMenuItems[activeIndex];
-      popover.dispatchEvent(getMockKeydownEvent(document.activeElement, CONSTANTS.KEYCODES.UP_ARROW_KEY));
+      popoverMenuItems[activeIndex].focus();
+      wrapper.instance().onKeyDown({
+        keyCode: CONSTANTS.KEYCODES.UP_ARROW_KEY,
```

Comment:
Could we keep the `getMockKeydownEvent` in order to avoid setting up the even to object each times. If the object changes in the future it will be easier to modify

Reply:
Will do